Makefile

```
@@ -91,7 +91,10 @@ package_statistics_collector: build_statistics_collector package_dir_structure
 	cd $(CC_BUILD_DIR) && \
 	ln -sf ../lib/python3/codechecker_statistics_collector/cli.py bin/post-process-stats
 
-package: package_dir_structure set_git_commit_template package_plist_to_html package_tu_collector package_report_converter package_report_hash package_merge_clang_extdef_mappings package_statistics_collector
+package_gerrit_skiplist:
+	cp -rp scripts/gerrit_changed_files_to_skipfile.py $(CC_BUILD_DIR)/bin
```

Do we need the `-r` option here? This option is needed for directories and this is just a file and not a directory.
```suggestion
	cp -p scripts/gerrit_changed_files_to_skipfile.py $(CC_BUILD_DIR)/bin
```